addons/project/static/src/components/project_status_with_color_selection/project_status_with_color_selection_field.js

```diff
@@ -24,6 +25,7 @@ export class ProjectStatusWithColorSelectionField extends SelectionField {
 ProjectStatusWithColorSelectionField.props = {
     ...SelectionField.props,
     statusLabel: { type: String, optional: true },
+    hideStatusText: { type: Boolean, optional: true },
```

Comment:
Shouldn't we just override StateSelectionField i/o SelectionField?

Reply:
Done!